fragments/k8s/cdk/README.md

```diff
@@ -495,11 +495,17 @@ documentation for more information.
  The following are known issues and limitations with the bundle and charm code:
 
  - Destroying the the easyrsa charm will result in loss of public key
- infrastructure (PKI).
+   infrastructure (PKI).
 
  - Deployment locally on LXD will require the use of conjure-up to tune
    settings on the host's LXD installation to support Docker and other
-   componentry.
+   components.
+
+ - If resources fail to download during initial deployment for any reason, you
+   will need to download and install them manually. For example, if
+   kubernetes-master is missing its resources, download them
+   [here](https://jujucharms.com/u/containers/kubernetes-master/) and install
```

Comment:
its not clear to an end user that the resources on the right hand side are what they are looking for. I wonder if there's more verbiage here we can use to guide them to look at that specific area of the store display.

Should we also link to the individual charms somewhere here in the readme so they don't have to guess/search where they need to go for resources?